libcnb/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Fix permission issues during layer handling when the layer contains read-only directories. ([#488](https://github.com/heroku/libcnb.rs/pull/488)).
 - Add explicit `DeleteLayerError` to provide more context when debugging layer handling problems. ([#488](https://github.com/heroku/libcnb.rs/pull/488)).
+- libcnb.rs now targets [Buildpack API 0.8](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.8). Buildpacks need to upgrade the `api` key to `0.8` in their `buildpack.toml`. ([#489](https://github.com/heroku/libcnb.rs/pull/489))
```

Comment:
Do we need to mention the new methods on `BuildResultBuilder` and `LayerResultBuilder` (given they are `libcnb` specific, not `libcnb-data` specific), or perhaps a brief mention of SBOM support?

Reply:
Yeah, I added an entry for those in [1a9ac2c](https://github.com/heroku/libcnb.rs/pull/489/commits/1a9ac2cec66aeac47398a4b9e89f8c47f009111b). 👍🏻